### ISSUE TITLE
poetry init should ask if the user wants to install dependencies after init

### ISSUE THREAD

**Issue by paullaffitte** [python-poetry/poetry#5456]
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] I have searched the [documentation](https://python-poetry.org/docs/) and believe that my question is not covered.

## Feature Request

The `poetry init` command is nice because it guides the user towards setting everything up while asking different questions. At the end of all those questions, including some about dependencies to add, poetry could ask the user if it should install the dependencies right away (with the default choice being "yes"). For me it feel like it's the last step before being able to do some work, so it would be nice to have it included in the init process.

tldr;
```bash
$ poetry init
# a few init steps later...
Would you like to install your dependencies now? (yes/no) [yes] 
```

**Comment by Rafeqm** [comment]
You can just can just run `poetry install --no-root`, right? I think there is no need to add features that is not very significant and also lengthened the core process of `poetry init`.

**Comment by paullaffitte** [comment]
> You can just can just run poetry install --no-root, right?

That's exactly the point of this issue, not having to write this line. If poetry already asks what dependencies to add to the project, it's quite obvious that the developer will have to run this command **every single time**  he ran the `init` one. So why not including this last step in the init process ? Currently the init process looks unfinished IMHO.
